kolena/workflow/test_run.py

```diff
@@ -401,6 +401,34 @@ def _iter_test_samples_batch(
             df_class=TestSampleDataFrame,
         )
 
+    def _extract_thresholded_metrics(
```

Comment:
Any chance this can be unit tested?

Reply:
unit test was added

Reply:
Is the plan to add integration test after backend PR?